### DISCUSSION TITLE
Can't connect to hyperdrive

### DISCUSSION THREAD

**Discussion by ceewanna** [beakerbrowser/beaker#1960] [Q&A]
I just try setting up a test hyperdrive and access it from another network. I found this page. Are there anything I need to set to make it accessible. 
![Screenshot from 2022-05-01 10-34-24](https://user-images.githubusercontent.com/84608596/166131085-185e38f0-c107-4df2-a037-d59bb55b6016.png)

**Comment by Nirei** [comment]
Having this exact same problem. Asked this question on #beakerbrowser in Libera IRC too but no answers.

**Reply by ceewanna** [reply]
Maybe the browser is not working and they have given up on it. I just want the p2p idea to communicate between 2 sites. Luckily I found [syncthing](https://syncthing.net/) and it works amazingly good for me.

**Reply by Nirei** [reply]
I'm guessing it's no longer capable of peer discovery. I'm trying to figure out why by reading docs. I was in the middle of writing a decentralized app for beaker and after a couple of weeks' work it's such a bummer that it doesn't work. Glad that you found something, though!

**Reply by Nirei** [reply]
Ok, I feel pretty stupid. In my case the problem was that I was behind CG-NAT. I called my ISP and asked to pay them 1€ more per month to get a real IP and it started working.

What kind of threw me off is that Beaker said "Hole punchable" the whole time... so I was a bit misled into thinking all was good.

**Reply by ceewanna** [reply]
My case I have both sides on a 4G router and the hole punchable turned on and off sometimes. With the same settings Syncthing can discover each other but BB can't see each other. I am an idiot user and just got to know p2p but it seems p2p requires discovery and relay server. Without them, each node seems be blind.
